tutorial/chapter/7/index.md

```diff
@@ -79,58 +43,58 @@ z = z;
 console.log(x, y, z);
 ```
 
-That is nonsense. In Squiggle, `let` and `def` bindings are not hoisted, and it
-is a runtime error to attempt to use their values before initialization.
+That is nonsense. In Squiggle, `let` and `def` bindings are not hoisted, and it is a runtime error to attempt to use their values before initialization.
 
 ## Destructuring
 
-The basic form of let is `let name = value`, but the `name` part can actually be
-shaped like an array or object, in order to pluck values from the right hand
-side.
-
-### Ignore
-
-The special identifier `_` can be used to ignore values. This is useful for
-evaluating an expression for its side effects, or ignoring values in arrays or
-objects.
+The basic form of let is `let name = value`, but the `name` part can actually be shaped like an array or object, in order to pluck values from the right hand side.
 
-```squiggle
-let _ = console.log("hi")
-in undefined
-```
+## Patterns
 
 ### Arrays
 
 Basically just write an array on the left, including a "..." before the last item to gobble up all remaining items. Number of items must match.
 
+Two item array:
+
 ```squiggle
 let [x, y] = [1, 2]
 #=> x == 1
 #=> y == 2
 ```
 
+Too few items:
+
 ```squiggle
 let [x, y] = [1]
 #=> error, too few items to unpack
 ```
 
+Too many items:
+
 ```squiggle
 let [x, y] = [1, 2, 3, 4]
 #=> error, too many items to unpack
 ```
 
+Rest array example 1:
+
 ```squiggle
 let [x, ...xs] = [1, 2, 3, 4]
```

Comment:
You currently have to write `[x, ..._]`, but I'm strongly considering adding `[x, ...]` as an alias for that case.